activerecord/lib/active_record/tasks/postgresql_database_tasks.rb

```diff
@@ -79,7 +79,7 @@ def structure_dump(filename, extra_flags)
 
       def structure_load(filename, extra_flags)
         args = ["--set", ON_ERROR_STOP_1, "--quiet", "--no-psqlrc", "--output", File::NULL, "--file", filename]
-        args.concat(Array(extra_flags)) if extra_flags
+        args = Array(extra_flags).concat(args) if extra_flags
```

Comment:
Instead of prepending the `extra_flags` afterwards, maybe start with the `extra_flags` and append the default flags?
```ruby
        args = Array(extra_flags)
        args.concat(["--set", ON_ERROR_STOP_1, "--quiet", "--no-psqlrc", "--output", File::NULL, "--file", filename])
```
Or:
```ruby
        args = Array(extra_flags) + ["--set", ON_ERROR_STOP_1, "--quiet", "--no-psqlrc", "--output", File::NULL, "--file", filename]
```

Reply:
I like the last option best. Will fix it today 👍

Reply:
@p8 All good! I tested it locally on my problematic PR as well, works like a charm.